File: data-quality/src/test/java/it/gov/daf/dataquality/analysis/email/checkers/ReservedCheckerTest.java
package it.gov.daf.dataquality.analysis.email.checkers;

import junit.framework.TestCase;
import org.junit.Test;

/**
 * @author Marco Bianchi (bianchi74@gmail.com)
 */
public class ReservedCheckerTest extends TestCase {

    @Test
    public void testIsChecked(){
        Checker checker = ReservedChecker.getInstance();
        assert(checker.isChecked("webmaster@"));
        assert(! checker.isChecked("notReserved@"));
    }
}

File: data-quality/trash/ReservedCheckerTest.scala
package it.gov.daf.dataquality.analysis.email.checkers

import org.scalatest.FunSuite

/**
  * @author Marco Bianchi (bianchi74@gmail.com)
  */
class ReservedCheckerTest extends FunSuite {

  test("getInstance"){
    ReservedChecker.getInstance();
  }

  test("testIsChecked") {
    val checker = ReservedChecker.getInstance()
    assert(checker.isChecked("webmaster@"));
    assert(! checker.isChecked("notReserved@"));
  }

}
